src/ring_theory/ideal/operations.lean

```diff
@@ -1604,3 +1611,65 @@ begin
 end
 
 end ring_hom
+
+namespace double_quot
+open ideal
+variables {R : Type u} [comm_ring R] (I J : ideal R)
+
+/-- define `quot_left_to_quot_sup` to be the obvious ring hom `R/I → R/(I ⊔ J)` -/
+def quot_left_to_quot_sup : I.quotient →+* (I ⊔ J).quotient :=
+ideal.quotient.factor I (I ⊔ J) (le_sup_left)
+
+/-- This will be used to lift `quot_left_to_quot_sup` to a map `(R/I)/J' → R/(I ⊔ J)`-/
+lemma ker_quot_left_to_quot_sup :
```

Comment:
This could be split up into smaller lemmas. I suggest proving that a general theorem about the kernel of `quotient.lift`. That should be enough to reduce the goal to proving `map (quotient.mk I) (I  ⊔ J) = map (quotient.mk I) J`. Then you could prove something like `map f I = map f J iff I ⊔ ker f = J ⊔ ker f` for a general ring hom and apply that theorem